### DISCUSSION TITLE
Fix The Function Creation with Curly Braces

### DISCUSSION THREAD

**Discussion by ramanvaidhya** [community/community#127424] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi Team,

I have been using Copilot for a while now and would like to share some feedback that I believe will significantly enhance the Copilot experience.

Whenever I write a new function or a conditional statement (such as an "if" statement), Copilot generates the name of the possible function only up to the opening brace. However, it does not automatically close the function brace or the conditional statement braces. This lack of automatic brace closure causes a lot of confusion, especially when working on existing code.

By addressing this issue, the workflow would become much smoother and more intuitive, reducing potential errors and increasing efficiency.

**Comment by ayim** [comment]
Hi @ramanvaidhya, which IDE are you using?

**Reply by ramanvaidhya** [reply]
Vs Code
